web/packages/teleport/src/Roles/RoleEditor/standardmodel.test.ts

```diff
@@ -45,6 +48,27 @@ const minimalRoleModel = (): RoleEditorModel => ({
   accessSpecs: [],
   rules: [],
   requiresReset: false,
+  options: {
+    maxSessionTTL: '30h0m0s',
+    clientIdleTimeout: '',
+    disconnectExpiredCert: false,
+    requireMFAType: {
+      value: false,
+      label: 'No',
+    },
+    createHostUserMode: {
+      value: '',
+      label: 'Unspecified',
+    },
+    createDBUser: false,
+    createDBUserMode: {
+      value: '',
+      label: 'Unspecified',
+    },
+    desktopClipboard: true,
+    createDesktopUser: false,
+    desktopDirectorySharing: true,
+  },
```

Comment:
The SAML IdP option is not included this time?

Reply:
We are supporting an arbitrary subset of options for now, so no, it's not included.